packages/dso-toolkit/components/Voorbeelden/modal-en-progress-indicator.njk

```diff
@@ -0,0 +1,21 @@
+{% render '@header', {menu: menu, breadcrumbs: []} -%}
+<body {{ className('dso-modal-open', [noMaxHeight, 'dso-scripted-height']) }}>
+<div class="dso-modal" tabindex="-1" role="dialog" aria-labelledby="modal-dialog">
+  <div class="dso-dialog" role="document">
+    <div class="dso-body">
+      <div class="dso-progress-indicator dso-progress-indicator-looping dso-small" role="progressbar" aria-valuetext="Een moment geduld alstublieft.">
```

Comment:
Hier kun je toch het component voor opnemen? https://dso-toolkit.nl/21.1.0/components/detail/progress-indicator.html

Dan hoef je zelf die HTML niet uit te typen (enfin, dat hoef ik niet uit te leggen denk ik).

Reply:
`{% render '@progress-indicator' %}` al dan niet met een data-objectje met de instellingen, of een stuk context dat je opneemt in je `modal-en-progress-indicator.config.yml`